tools/spec-gen-sdk/package.json

```
@@ -5,7 +5,7 @@
     "email": "azsdkteam@microsoft.com",
     "url": "https://github.com/Azure/azure-sdk-tools"
   },
-  "version": "0.6.1",
+  "version": "0.6.2",
```

Lets revert before merging
```suggestion
  "version": "0.6.1",
```